agent/04_agent_configure.sh

```diff
@@ -268,6 +268,7 @@ else
     ip=${AGENT_NODES_IPSV6[0]}
   fi
   configure_dnsmasq ${ip} ""
+  echo "${ip}" > ${OCP_DIR}/node0-ip
```

Comment:
This is in the section that only runs for SNO clusters, so that's likely why the compact jobs are failing.

Reply:
With https://github.com/openshift-metal3/dev-scripts/pull/1488 we don't need this now.